tests/webtests/EntityPublishToTestTest.php

```diff
@@ -101,6 +101,8 @@ public function test_it_published_metadata_to_manage()
     {
         // Entity id validation
         $this->testMockHandler->append(new Response(200, [], '{"id":"f1e394b2-08b1-4882-8b32-43876c15c743"}'));
+        // Unique validation
```

Comment:
Why not add a test case where you test the correct integration of the uniqe enitty id validator? This would be a great addition to thet test arsenal!

Reply:
Good suggestion, done